opentelemetry-sdk/src/resource/mod.rs

```diff
@@ -62,6 +62,24 @@ impl Default for Resource {
 }
 
 impl Resource {
+    /// Creates a Builder that allows you to configure multiple aspects of the Resource.
+    ///
+    /// Starts with a [Resource::default()].
```

Comment:
nit: It would be nice to list out the default resources that would be added for easy reference.

Reply:
updated